pkgs/development/libraries/science/math/scalapack/default.nix

```
@@ -73,7 +74,7 @@ stdenv.mkDerivation rec {
     homepage = "http://www.netlib.org/scalapack/";
     description = "Library of high-performance linear algebra routines for parallel distributed memory machines";
     license = licenses.bsd3;
-    platforms = [ "x86_64-linux" ];
-    maintainers = with maintainers; [ costrouc markuskowa ];
+    platforms = [ "x86_64-linux" "x86_64-darwin" "aarch64-darwin" ];
```

Would you mind trying
```suggestion
    platforms = platforms.unix;
```
?
If OfBorg can build it, this would be more straight forward.

---

done